DP02_16_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,1566 @@
+{
```

Comment:
<p>Suggested: </p><p> </p><p>""There is a concentration of galaxies at <strong>g-r ≈ 1.1</strong>, corresponding to galaxies with varying brightness in the <strong>r</strong>-band magnitude. This concentration represents the <strong>red sequence (RS)</strong>. To select background galaxies, choose those that are redder than the RS while allowing for a buffer to account for magnitude measurement uncertainties."</p><p> </p><p>Also: <code>g</code> , <code>r</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>adjusted</p>